publish.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+JOB_TYPE=${JOB_TYPE:-local}
+REPO_OWNER=${REPO_OWNER:-openshift}
+REPO_NAME=${REPO_NAME:-dynamic-plugin-sdk}
+PULL_NUMBER=${PULL_NUMBER:-0}
+NPM_TOKEN=${NPM_TOKEN:-notoken}
+FORCE_PUBLISH=${FORCE_PUBLISH:-false}
+
+LABEL_PUBLISH="publish-sdk"
+PACKAGE_GLOB=(./packages/lib-*)
+NPM_REGISTRY="https://registry.npmjs.org/"
+NPM_REGISTRY_AUTH_KEY="//registry.npmjs.org/:_authToken"
+LABELS=$(curl -s \
```

Comment:
This triggers HTTP call to GitHub API even if we run the script locally as part of manual `test.sh` execution.

Can we make this HTTP call conditional, i.e. use an empty array if `PULL_NUMBER` is not `0` ?